SimplenoteScreenshots/SimplenoteScreenshots.swift

```diff
@@ -135,36 +129,31 @@ class SimplenoteScreenshots: XCTestCase {
         openMenu(using: app)
         loadPasscodeScreen(using: app)
         // Set the passcode
-        // Writing the value here in clear because anyways one can see it being typed.
-        type("1234", onFirstKeyboardOf: app)
+        // Writing the value here in clear because one can see it being typed anyways.
+        typeOnPassCodeScreen(1234, using: app)
```

Comment:
The passcode screen now has custom buttons instead of loading the iOS keyboard.

Reply:
Hey – this is the passcode to my luggage!